packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -133,17 +133,20 @@ export class SliceMachineInitProcess {
 					)}\n`,
 				);
 			}
-			await this.manager.telemetry.initTelemetry({
-				appName: pkg.name,
-				appVersion: pkg.version,
-			});
+
 			try {
 				await setupSentry();
```

Comment:
Do you think we could put Sentry even higher up? 

I would go the highest place in this function, WDYT?